dscreator/datasets/trajectories/ferrybox.py

```diff
@@ -117,11 +139,23 @@ def variable_attributes(self, variable_name: str):
                 return attrs
             case "oxygen_qc":
                 return asdict(FlagAttrs(long_name="Sea Water Oxygen Saturation Quality Flag"))
-
-            case "cdom":
-                return asdict(
-                    CFVariableAttrs(standard_name="", long_name="", units="mg/m^3"),
-                )
             case _:
                 logging.warning(f"Array definition not found for: {variable_name}")
                 raise RuntimeError(f"Array definition not found for: {variable_name}")
+
+
+@dataclass
+class DailyFantasy(NorsoopFantasy):
+    def dataset_attributes(self, ds: xr.Dataset) -> FerryboxDatasetAttrs:
+        attrs = super().dataset_attributes(ds)
+        attrs.title = "Ferrybox on MS Color Fantasy, daily data"
+        attrs.title_no = "Ferje på MS Color Fantasy, daglige data"
+        attrs.summary = "Daily updating dataset of ferrybox data for ferry sailing Oslo, Norway to Kiel, Germany. For more information see: https://www.colorline.no/oslo-kiel/fakta."
+        attrs.summary_no = "Daglig updatert dataset fra ferrybox ombord ferje fra Oslo, Norge til Kiel, Tyskland. For mer informasjon se: https://www.colorline.no/oslo-kiel/fakta."
+        attrs.keywords += "," + ",".join(
+            [
+                "GCMDSK:EARTH SCIENCE > OCEANS > OCEAN CHEMISTRY > CHLOROPHYLL",
```

Comment:
They have 
EARTH SCIENCE > OCEANS > OCEAN OPTICS > FLUORESCENCE
Can we create 
EARTH SCIENCE > OCEANS > OCEAN OPTICS > CHLOROPHYLL? 

IN the same way as they have 
EARTH SCIENCE > OCEANS > OCEAN OPTICS > TURBIDITY

Reply:
Can check it comes from here https://gcmd.earthdata.nasa.gov/KeywordViewer